dist/steps/run_tests.sh

```diff
@@ -88,6 +108,28 @@ for platform in ${TEST_PLATFORMS//;/ }; do
   # Print unity log output
   cat "$FULL_ARTIFACTS_PATH/$platform.log"
 
+  if [[ $TEST_EXIT_CODE -eq 0 && "$platform" == "standalone" ]]; then
+    echo ""
+    echo "###########################"
+    echo "#    Testing Standalone   #"
+    echo "###########################"
+    echo ""
+
+    # Not sure how to get code coverage with this method.
+    xvfb-run -a -e /dev/stdout "$UNITY_PROJECT_PATH/Build/UnityTestRunner-Standalone" \
+      -batchmode \
+      -nographics \
+      -logFile "$FULL_ARTIFACTS_PATH/$platform-player.log" \
+      -testResults "$FULL_ARTIFACTS_PATH/$platform-results.xml"
+
+    # Catch exit code
+    TEST_EXIT_CODE=$?
+
+    # Player log gets spammed with connection failed messages from the PlayerConnection,
+    # so we don't print it here. If there was an issue with the player tests, users can inspect
+    # the log from the arifacts.
```

Comment:
Is this something we should be creating an issue for, or is this a Unity Editor issue?

Reply:
That's a Unity PlayerConnection issue. I searched the logs for a way to disable it explicitly, but such a method does not exist. It does eventually disable itself when enough failed connections occur, but it's a lot on Linux (not so many on Windows, no idea why the difference).

Reply:
I may be able to manually strip those connection failed messages out of the log to print it reasonably if you think that's a good idea.

Reply:
Ah nevermind, I figured it out. Just needed to disable it in the build options.

```cs
// Do not launch the player after the build completes. Disable the PlayerConnection.
playerOptions.options &= ~(BuildOptions.AutoRunPlayer | BuildOptions.ConnectToHost | BuildOptions.WaitForPlayerConnection);
```